dist/function.py

```diff
@@ -229,12 +229,19 @@ def infer_module(path):
             print(f'{path}: Unexpected path')
             sys.exit(1)
 
-        # If the path is in src, the module is the subdirectory under src; otherwise it is the
-        # top-level directory, such as "test."
+        # If the path is in src, the module is the subdirectory under src.
+        # If the path is test/unittest, the module is unittest.
+        # Otherwise it is the top-level directory, such as "test."
         if parts[1] == 'src':
             module = parts[2]
+        elif parts[1] != "test":
+            module = parts[1]
+        elif parts[2] == "unittest":
+            # Treat test/unittests special since they are allowed to call __wti_ functions.
+            module = parts[2]
         else:
             module = parts[1]
+            
```

Comment:
Unnecessary blank line.

Reply:
I minor point: a single blank line (already present) between these sections of code is good as it makes the code easier to read. As @mm-ng points out a second blank line is unnecessary.

Reply:
Done. This was an unintentional editing error.